packages/amplify_authenticator/lib/src/widgets/authenticator_banner.dart

```diff
@@ -79,4 +77,17 @@ extension on StatusType {
         return Icons.error;
     }
   }
+
+  ColorSwatch<int> get color {
+    switch (this) {
+      case StatusType.info:
+        return Colors.blue;
+      case StatusType.success:
+        return Colors.green;
+      case StatusType.warning:
```

Comment:
Is `StatusType.warning` used anywhere? if not, can you remove it?

Reply:
Good call. I don't think warning or success get used currently.